roles/wazuh/ansible-wazuh-agent/defaults/main.yml

```diff
@@ -16,7 +16,6 @@ wazuh_agent_sources_installation:
   user_language: "y"
   user_no_stop: "y"
   user_install_type: "agent"
-  user_dir: "/var/ossec"
```

Comment:
Hi @sergiogp98,

This variable is related to sources installation, I think it would a good idea to keep it despite it is currently unused.